### PR TITLE
Update for Kedro 0.17.0

### PR DESCRIPTION
**Pull request by lorenabalan**
Updating this ahead of beginner training this week.

Steps:

```
kedro new --starter spaceflights
```

prompts:

```
kedro_tutorial
spaceflight
kedro_tutorial
```

Deleted pipeline code from `hooks.py`, `src/pipelines/data_engineering/pipeline.py` and `src/pipelines/data_science/pipeline.py`.

### CODE REVIEW

File: kedro-exercises/spaceflight/setup.cfg [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+line_length=88
+known_third_party=kedro
+
 [tool:pytest]
 addopts=--cov-report term-missing
         --cov src/kedro_tutorial -ra
```
Comment (andrii-ivaniuk) [inline kedro-exercises/spaceflight/setup.cfg:15]: `isort` and `pytest` are redundant here. They are configured in `toml` file

File: kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_engineering/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+This modular pipeline preprocesses the raw data (`preprocessing_companies` and `preprocessing_shuttles` nodes) and then creates the master table (`master_table` node).
+
+## Pipeline inputs
+
+### `companies`
+
+|      |                    |
+| ---- | ------------------ |
+| Type | `pandas.DataFrame` |
+| Description | Source data about the companies provide the space shuttles |
```
Comment (andrii-ivaniuk) [inline kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_engineering/README.md:16]:
```suggestion
| Description | Source data about the companies that provide the space shuttles |
```
?

File: kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_engineering/__init__.py [hunk 3]
```diff
@@ -1 +1,31 @@
+# Copyright 2020 QuantumBlack Visual Analytics Limited
```
Comment (andrii-ivaniuk) [inline kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_engineering/__init__.py:1]:
```suggestion
# Copyright 2021 QuantumBlack Visual Analytics Limited
```
I think you need to update this in all files.

File: kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_science/__init__.py [hunk 4]
```diff
@@ -1 +1,31 @@
+# Copyright 2020 QuantumBlack Visual Analytics Limited
```
Comment (andrii-ivaniuk) [inline kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_science/__init__.py:1]:
```suggestion
# Copyright 2021 QuantumBlack Visual Analytics Limited
```

File: kedro-exercises/spaceflight/src/kedro_tutorial/settings.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 QuantumBlack Visual Analytics Limited
```
Comment (andrii-ivaniuk) [inline kedro-exercises/spaceflight/src/kedro_tutorial/settings.py:1]:
```suggestion
# Copyright 2021 QuantumBlack Visual Analytics Limited
```

File: kedro-exercises/spaceflight/src/requirements.txt [hunk 6]
```diff
@@ -5,7 +5,7 @@ isort>=4.3.21, <5.0
 jupyter~=1.0
 jupyter_client>=5.1,<7.0
 jupyterlab==0.31.1
-kedro[pandas.CSVDataSet,pandas.ExcelDataSet]==0.16.6
+kedro[pandas.CSVDataSet,pandas.ExcelDataSet]==0.17.0
```
Comment (andrii-ivaniuk) [inline kedro-exercises/spaceflight/src/requirements.txt:8]: Does it use `pandas.ExcelDataSet` dataset?

File: kedro-exercises/spaceflight/src/setup.py [hunk 7]
```diff
@@ -52,14 +54,15 @@
         "docs": [
```
Comment (andrii-ivaniuk) [inline kedro-exercises/spaceflight/src/setup.py:54]: Could you sync `docs` dependencies with `kedro` code? Some deps are redundant and some need a version update.

File: kedro-exercises/spaceflight/src/requirements.txt (hunk 6 above)
Comment (lorenabalan) [reply kedro-exercises/spaceflight/src/requirements.txt:8]: Yup the `shuttles` dataset uses it.

File: kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_engineering/__init__.py (hunk 3 above)
Comment (lorenabalan) [reply kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_engineering/__init__.py]: Done.

File: kedro-exercises/spaceflight/setup.cfg (hunk 1 above)
Comment (lorenabalan) [reply kedro-exercises/spaceflight/setup.cfg]: Fixed.

File: kedro-exercises/spaceflight/src/setup.py (hunk 7 above)
Comment (lorenabalan) [reply kedro-exercises/spaceflight/src/setup.py:54]: I think these are in line with the requirements packaged with kedro 0.17.0 at the time, no?

**Review by yetudada — APPROVED** [review]
🏆

File: kedro-exercises/spaceflight/src/setup.py (hunk 7 above)
Comment (andrii-ivaniuk) [reply kedro-exercises/spaceflight/src/setup.py:54]: Oh, sorry I missed that it is going to be updated in the future release,

**Review by andrii-ivaniuk — APPROVED** [review]

**Pull request merged**